### PR TITLE
New UI for editor sidebar & bottom bar

### PR DESCRIPTION
**Pull request by robertu7**
* New UI for editor sidebar (desktop) and bottom bar (mobile);
  * Use `<SearchSelect>`;
  * Support editing cover;
  * Remove legacy sidebar components;
* Add support to create a new tag on `<SearchSelect>`;
* Add `useImperativeQuery` for lazy loading with Promise supports;

### CODE REVIEW

File: src/views/ArticleDetail/EditMode/index.tsx [hunk 1]
```diff
@@ -1,7 +1,155 @@
-import Header from './Header'
```
Comment (robertu7) [inline src/views/ArticleDetail/EditMode/index.tsx:1]: I moved EditMode related code to this file, please notice. @devformatters

File: src/components/Editor/SetCoverDialog/Uploader.tsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  entityId: string
+  entityType: ENTITY_TYPE.article | ENTITY_TYPE.draft
+}
+
+type UploaderProps = {
+  setSelected: (asset: Asset) => any
+  refetchAssets: () => any
+} & UploadEntity
+
+const Uploader: React.FC<UploaderProps> = ({
```
Comment (devformatters) [inline src/components/Editor/SetCoverDialog/Uploader.tsx:32]: Is it possible to reuse `~/components/FileUploader` ?

**Review by devformatters — APPROVED** [review]
Nice work. 🔋

File: src/components/Editor/SetCoverDialog/Uploader.tsx (hunk 2 above)
Comment (robertu7) [reply src/components/Editor/SetCoverDialog/Uploader.tsx:32]: `FileUploader/*` have two UI for avatar and profileCover uploading, so seems doesn't fit in this case.

**Pull request merged**